RealmSwift-swift1.2/RealmConfiguration.swift

```diff
@@ -147,10 +147,12 @@ extension Realm {
 
         internal var rlmConfiguration: RLMRealmConfiguration {
             let configuration = RLMRealmConfiguration()
-            if self.path != nil {
+            if path != nil {
```

Comment:
I just copy pasted it from the swift-2.0 case, where `self.` wasn't included in the `path` check. In general, I prefer to omit `self` unless required (some rationale [here](https://github.com/github/swift-style-guide#only-explicitly-refer-to-self-when-required)).